src/core/Akka.TestKit/TestKitBase.cs

```diff
@@ -710,10 +718,35 @@ public IActorRef CreateTestActor(string name)
             return CreateTestActor(_testState.System, name);
         }
 
+        private IActorRef CreateInitialTestActor(ActorSystem system, string name)
+        {
+            var testActorProps = Props.Create(() => new InternalTestActor(_testState.Queue))
+                .WithDispatcher("akka.test.test-actor.dispatcher");
+            
+            var testActor = system.AsInstanceOf<ActorSystemImpl>().SystemActorOf(testActorProps, name);
+            
+            // Force the TestActor to start synchronously to avoid deadlock
```

Comment:
Took a different tact after the test failures from the first fix, caused by the `TestActor` being incorrectly registered to the `/system` guardian actor. This time around we just force the `CallingThreadDispatcher` to run by sending a dummy message to the `TestActor` - this should cause the queued `Supervise` system message for the `TestActor` to get processed before we even start waiting for test initialization.